lib/paperclip/storage/eitheror.rb

```diff
@@ -73,7 +73,7 @@ def usable_storage
         either_exists = @either.exists?
         or_exists = @or.exists?
 
-        return @either if either_exists || !or_exists
+        return @either if !or_exists || either_exists
```

Comment:
@mpabegg I did not run any benchmarks for this change, it just feels to me that'd be the case (see commit body).

Thoughts?